recipes-kernel/linux/files/netfilter.cfg

```diff
@@ -205,6 +213,7 @@ CONFIG_NFT_REJECT_IPV4=m
 CONFIG_NFT_DUP_IPV4=m
 CONFIG_NFT_FIB_IPV4=m
 CONFIG_NF_TABLES_ARP=y
+# CONFIG_NF_TABLES_BRIDGE is not set
```

Comment:
disabled on purpose?

Reply:
I double checked that and found that I misinterpreted what make menuconfig tells me about config options.
so, this option is now active and a few more were added, pls review again.